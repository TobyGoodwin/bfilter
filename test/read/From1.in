From fakefrom
Dull: header
Subject: foo

We want to check that we can start a line with the word
from or even a blank line followed by

from me to you, and all is well.

>From of course is escaped like so

And that's all!

From fakefrom2
Dull: heade

dXNmY2EuZWR1PiB3cm90ZToKPiBIaS4gSnVzdCBjdXJpb3VzLiBIYXMgYW55b25lIGZpZ3VyZWQg
