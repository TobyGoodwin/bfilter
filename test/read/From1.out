{ foo}
{We want to check that we can start a line with the word}
{from or even a blank line followed by}
{from me to you, and all is well.}
{>From of course is escaped like so}
{And that's all!}
{usfca.edu> wrote:
> Hi. Just curious. Has anyone figured }
